.devcontainer/setup.sh

```diff
@@ -8,8 +8,8 @@ export PS1='\[\e[3;36m\]\w ->\[\e[0m\\] '
 # Update Nextflow
 nextflow self-update
 
-# Install pre-commit hooks
-pre-commit install --install-hooks
+# Install prek hooks
+prek install --install-hooks
```

Comment:
Is this installed in the container yet? 
Running this PR in a devcontainer fails silently. It starts, but prek is unavailable.

```suggestion
pip install prek
prek install --install-hooks
```